matter/efr32/efr32mg24/BRD4186C/autogen/sl_event_handler.c

```diff
@@ -31,7 +31,10 @@
 #ifdef SL_WIFI
 #include "sl_spidrv_instances.h"
 #endif
-
+#ifdef CHIP_917
+#include "btl_interface.h"
```

Comment:
you removed the init call but not the include. You can remove those two also

Reply:
Removed